lib/commands/project/project-command-base.ts

```diff
@@ -3,15 +3,20 @@
 
 export class ProjectCommandBase {
 	constructor(protected $errors: IErrors,
+		protected $nameCommandParameter: ICommandParameter,
 		protected $project: Project.IProject) { }
 
-	protected canExecuteCore(): IFuture<boolean> {
+	protected canExecuteCore(args: string[]): IFuture<boolean> {
 		return (() => {
+			if (args.length > 1) {
+				this.$errors.failWithoutHelp("This command accepts only one argument.");
```

Comment:
You should fail with help here. The user must understand command's syntax from the help.